test/regexp.test.js

```diff
@@ -27,28 +28,31 @@ describe('cloudant regexp', function() {
   });
   it('find all foos begining with b', function(done) {
     Foo.find({where: {bar: {regexp: '^b'}}}, function(err, entries) {
-      console.log (entries);
+      if (err) return done(err);
+      console.log(entries);
       entries.should.have.lengthOf(1);
       entries[0].bar.should.equal('b');
       done();
     });
   });
   it('find all foos like b', function(done) {
     Foo.find({where: {bar: {like: 'b'}}}, function(err, entries) {
-      console.log (entries);
+      if (err) return done(err);
+      console.log(entries);
       entries.should.have.lengthOf(1);
       entries[0].bar.should.equal('b');
       done();
     });
   });
   it('find all foos not like b', function(done) {
     Foo.find({where: {bar: {nlike: 'b'}}}, function(err, entries) {
-      console.log (entries);
-      entries.should.have.lengthOf(N-1);
+      if (err) done(err);
+      console.log(entries);
```

Comment:
BTW, we should actually remove the console log line altogether. But that is outside the scope of this PR.